src/main/java/step3/domain/Lotto.java

```diff
@@ -0,0 +1,27 @@
+package step3.domain;
+
+import step3.domain.number.LottoMatchingCount;
+import step3.domain.number.LottoNumber;
+import step3.domain.number.LottoNumbers;
+
+public class Lotto {
+  private final LottoNumbers lottoNumbers;
+
+  public Lotto(LottoNumbers lottoNumbers) {
+    this.lottoNumbers = lottoNumbers;
+    lottoNumbers.sort();
+  }
+
+  @Override
+  public String toString() {
+    return lottoNumbers.toString();
+  }
+
+  public LottoMatchingCount matchLotto(Lotto prizeLotto) {
+    return lottoNumbers.matchNumbers(prizeLotto.lottoNumbers);
+  }
```

Comment:
> has-a(의존)에 대해 말씀해주셨던 부분에 대해, enum으로 옮겨가면서 사실 LottoMatchingNumber의 의미가 크게 줄었고, 이에 따라. 내부에서만 데이터 전송용으로 사용되어 불필요하게 존재하게 되나 의미를 구분하기 위해 놓아 어떻게 해야할지 고민되어 일단 두었습니다

enum으로 옮겨주셨군요 👍 👍 
말씀해주신대로 LottoMatchingCount는 크게 의미가 없어서 Count만으로도 의미 전달이 가능하기에
제거해주시면 좋을 것 같아요 :)

```suggestion
  public Count matchLotto(Lotto prizeLotto) {
    return lottoNumbers.matchNumbers(prizeLotto.lottoNumbers);
  }
```

has - a 관계는
```
class Count {
    private LottoMatchingCount count;
}
```
로 매칭 카운트라는 의미로만 생각해보았는데, 다시 보니 Count라는 객체는 로또 복권의 갯수도 될 수 있었네요..!
has - a 관계 피드백은 잊어주시면 좋을 것 같아요 ㅎㅎ; 🙏

Reply:
자세한 샘플 감사합니다! 헷갈렸는데 무슨 의미로 말씀하셨는지 확실히 다가오네요 ㅜㅜ 👍